### PR TITLE
Typescript Debugging and hidding .js + .js.map 

### PR DESCRIPTION
**Pull request by skjr001**
Here are the changes I have been talking about, sorry for taking so long.  I ran the tests in the project and everything passed 100%.  Take a look and let me know if you have any issues or questions. 

SK

### CODE REVIEW

**Comment by beverts312** [comment]
Looks good to me, glad you got the sourcemaps working.  
  
@calebcartwright Tests passed locally for me on windows (Mac tests fail even with master, I will open a bug if I can confirm its not something weird in my environment)

File: generators/app/templates/vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "files.exclude": {
+        "**/src/**.js": true,
```
Comment (calebcartwright) [inline generators/app/templates/vscode/settings.json]: @skjr001 does './src/**/*.js' here cover the entire directory?

**Comment by calebcartwright** [comment]
also I'm not sure why the circleci build appears to have frozen, but I will override and merge when ready

**Comment by beverts312** [comment]
@calebcartwright does this conflict with your changes?

**Pull request merged**